### PR TITLE
Add hash to frontend assets to clear browser caches between versions

### PR DESCRIPTION
**Pull request by niekcandaele**
Wanted to do hashing originally but couldn't get the sails Grunt pipeline to agree with me. Sails is supposed to have support for it but when generating a new sails app, there's no hashes in the prod build of assets :)

Adding the version number to the files should be good enough

### CODE REVIEW

File: views/layouts/homepageLayoutNoFooter.ejs [hunk 1]
```diff
@@ -128,24 +128,24 @@
...
-  <script src="/js/sdtdServer/chat.js"></script>
-  <script src="/js/sdtdServer/console.js"></script>
-  <script src="/js/sdtdServer/map.js"></script>
+  <script src="/dependencies/sails.io.js?v=1.17.5"></script>
+  <script src="/dependencies/jquery-3.2.1.js?v=1.17.5"></script>
+  <script src="/dependencies/jquery-3.2.1.min.js?v=1.17.5"></script>
+  <script src="/dependencies/bootstrap/js/bootstrap.bundle.js?v=1.17.5"></script>
+  <script src="/dependencies/bootstrap/js/bootstrap.bundle.min.js?v=1.17.5"></script>
+  <script src="/dependencies/bootstrap/js/bootstrap.js?v=1.17.5"></script>
+  <script src="/dependencies/bootstrap/js/bootstrap.min.js?v=1.17.5"></script>
```
Comment (halkeye) [inline views/layouts/homepageLayoutNoFooter.ejs:137]: why are all these hard coded to 1.17.5?
why not just add it to the configs/views.js's locales section and then <%= everywhere

Comment (halkeye) [reply views/layouts/homepageLayoutNoFooter.ejs:137]: oh cause it does a build. doing it with builds will cause it to make prs messier and stuff won't it?

Comment (niekcandaele) [reply views/layouts/homepageLayoutNoFooter.ejs:137]: This is autogenerated by sails/grunt, anything between `<!--STYLES-->` and `<!--STYLES END-->`, same for scripts. The `sails-linker` task is responsible for that

Comment (halkeye) [reply views/layouts/homepageLayoutNoFooter.ejs:137]: /tableflip
kill it with fire

its going to make releasing so messy

Comment (niekcandaele) [reply views/layouts/homepageLayoutNoFooter.ejs]: Yep.. :/ The version in the repo will always lag one behind. I'll give hashes another go, if we hash the file content then this auto generated crap will only change when the assets change.

**Comment by halkeye** [comment]
@niekcandaele feel free to `git revert 397f18a8` if you dislike the hashing, but i figured i could do it :)

**Comment by halkeye** [comment]
Made it even more rails like with script and stylesheet tag functions.
To undo, `git reset dc7f1604b651044ca40e6764de4017afcd6a7310`  then force push

**Comment by niekcandaele** [comment]
Beautiful! Thanks man

**Pull request merged**